src/bucket.ts

```diff
@@ -79,6 +79,21 @@ interface WatchAllOptions {
   versions?: boolean;
 }
 
+/**
+ * @typedef {object} AddLifecycleRuleOptions Configuration options for Bucket#addLifecycleRule().
+ * @property {string} [append=true] The new rules will be appended to any
```

Comment:
@stephenplusplus could you provide an example of how this works? It does reduce the development lifecycle of removing all rules and adding a new rule to 1 request instead of 2.

Reply:
I just added an example here: https://github.com/googleapis/nodejs-storage/pull/471/files#r228275243 -- is that something like what you were thinking?